app/models/user.rb

```diff
@@ -1,4 +1,9 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable, :confirmable
+  # Include default devise modules. Others available are
```

Comment:
- [ ] Hey there! 🚀 Great effort on updating the User model with :confirmable, but it seems there's a missing piece. 😅 Without a corresponding migration, the database won't reflect these changes, which can lead to issues.

### # To help you out
 - [ ] I found a fantastic resource 📺 that walks you through adding :confirmable to your Devise User model and creating the necessary migration. Check out this video tutorial: [Adding Confirmable to Devise User Model](https://www.youtube.com/watch?v=GXsL2p0YoHg). It should guide you through the process step by step.

#### Keep up the good work, and don't hesitate to ask if you have any questions along the way! 🌟💡😊